### ISSUE TITLE
Add infrastructure health instead of state

### ISSUE THREAD

**Issue by zweck** [pjhauser/odysseus#22]
All infrastructure items have an on/off state, this should be switched to a health percentage instead.

**Comment by yogaraptor** [comment]
Ooh, might we want to keep on/off for an "enable/disable" concept? E.g. the bridge could be disabled in that you can't access it, even though it's in perfect condition.

**Comment by zweck** [comment]
Ok, I thought you'd just set the health to 0 to disable but if you can
disable/eneable it I suppose you can then do things like block the entrance
and the infrastructure just greys out in the UI.

On Thu, Oct 22, 2015 at 2:35 PM, Tom Jenkins notifications@github.com
wrote:

> Ooh, might we want to keep on/off for an "enable/disable" concept? E.g.
> the bridge could be disabled in that you can't access it, even though it's
> in perfect condition.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/pjhauser/odysseus/issues/22#issuecomment-150225318.

## 

_Phil Hauser_
pjhauser.com
07912934267

**Comment by yogaraptor** [comment]
Yeah, cause if the entrance is blocked you need to be able to have the room be in the same health when the entrance is unblocked. Otherwise it's like those speakers with the volume controls that double as an on/off switch. Man I hated those.

**Comment by zweck** [comment]
yeh, thats cool. So it just needs adding as a prop instead :) easy peesy.

On Thu, Oct 22, 2015 at 2:44 PM, Tom Jenkins notifications@github.com
wrote:

> Yeah, cause if the entrance is blocked you need to be able to have the
> room be in the same health when the entrance is unblocked. Otherwise it's
> like those speakers with the volume controls that double as an on/off
> switch. Man I hated those.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/pjhauser/odysseus/issues/22#issuecomment-150227378.

## 

_Phil Hauser_
pjhauser.com
07912934267
